bravado/swagger_model.py

```diff
@@ -115,13 +115,13 @@ def load_yaml(self, text):
         :raise: yaml.parser.ParserError: If the text is not valid YAML.
         """
         data = yaml.safe_load(text)
-        for path, methods in iter(data.get('paths', {}).items()):
-            for method, operation in iter(methods.items()):
+        for path, methods in iter(list(data.get('paths', {}).items())):
+            for method, operation in iter(list(methods.items())):
                 if 'responses' in operation:
                     operation['responses'] = dict(
                         (str(code), response)
                         for code, response in iter(
-                            operation['responses'].items()
+                            list(operation['responses'].items())
```

Comment:
operation['responses'].items() is iterable, there is not need to change it to list and make it iterable with iter, 
**for code, response in operation['responses'].items() ** and same for above expressions at lines 118, 119.

Reply:
done